tests/test_sentinel2_processing_commands.py

```diff
@@ -95,6 +96,45 @@ def test_download_import_commands(self):
 
         # self.assertTrue(len(result) == 2)
 
+    def test_download_import_commands_noquery(self):
```

Comment:
Isn't this a unit test?

Reply:
Yes, you are right. The `pytest` decorator has been added and it is moved to the `unittest` dir